lib/streamer.js

```diff
@@ -406,19 +406,13 @@ class NexusStreamer extends EventEmitter {
     if (sessionInfo) {
       let width = 1280;
       let height = 720;
-      let fps = 30;
-      let bitrate = 300;
+      let bitrate = "1500k";
```

Comment:
original bitrate is `300k`. HomeKit is really sensitive about this and fps, especially during remote access.

Reply:
And there is already a "k" in the ffmpeg command...

Reply:
Oh ok interesting...

Sorry I missed that in the original cmd string string 🤦‍♂️